### PR TITLE
Update edge authentication proposal about cert rotation

### PR DESCRIPTION
**Pull request by XJangel**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test
> /kind feature

/kind design

**What this PR does / why we need it**:
add cert rotation in edge authentication proposal

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: docs/proposals/edge-authentication.md [hunk 1]
```diff
@@ -113,9 +113,39 @@ Then the CLoudCore and EdgeCore can establish mutual authentication TLS.
...
+The old certificate is replaced by new certificate. The EdgeCore reconnects with CloudCore.
+
+**step 5:** 
+
+Then the EdgeCore waits for the next rotation.
+
 
 
 ## Future development tasks
 1. Ports integration.
```
Comment (fisherxu) [inline docs/proposals/edge-authentication.md:151]: 2. Save the token to a file, delete the token file after the edge node is successfully registered.

Comment (XJangel) [reply docs/proposals/edge-authentication.md:151]: Got it !

**Pull request merged**